egov-workflow-v2/src/main/resources/db/migration/main/V20201030131738__wf_comment_size_ddl.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE eg_wf_processinstance_v2 ALTER COLUMN comment type character varying(1024);
+CREATE INDEX idx_tenant_status_eg_wf_processinstance_v2 ON eg_wf_processinstance_v2 USING btree ((tenantid || ':' || status));
```

Comment:
Always use `CREATE INDEX IF NOT EXISTS`

Reply:
I cannot change it now as flyway migration has already ran on dev and qa env